packages/assets-controllers/src/NftController.test.ts

```diff
@@ -1256,7 +1259,7 @@ describe('NftController', () => {
         tokenId: '1',
         favorite: false,
         isCurrentlyOwned: true,
-        tokenURI: '',
```

Comment:
I wasn't able to determine where this was coming from. The `NftController` always sets this to `null` if it's not found. Probably this was coming from the `AssetsContractController` as well.

Reply:
Ah, figured it out. It was from this line: https://github.com/MetaMask/core/blob/9687332f907e19a167a097f35f582678cc2e3859/packages/assets-controllers/src/NftController.ts#L571